EpicLoot/Patching/FilePatching.cs

```diff
@@ -203,7 +224,7 @@ public static string GetPatchesDirectoryPath()
                 patchesFolderPath = Path.Combine(Path.GetDirectoryName(assembly.Location) ?? string.Empty, "patches");
                 if (!Directory.Exists(patchesFolderPath))
                 {
-                    return null;
+                    Directory.CreateDirectory(patchesFolderPath);
```

Comment:
Why create the folder here if it's empty?

Reply:
So, this is to provide the folder that people SHOULD put patches, instead of trying to instruct them to create that specific folder themselves. 

It's harmless when empty, and is a QoL in my opinion